### DISCUSSION TITLE
Generating widgets throwing error

### DISCUSSION THREAD

**Discussion by chaseconey** [filamentphp/filament#12932] [Help]
### Package

Widgets

### Package Version

v3.2.82

## Overview

Hello there 👋 .

I have a fairly standard installation and setup AFAICT. I am using the latest version of `filament/filament` and when I am generating a widget to be added to a resource, I am getting the following error:

```
Unable to find component: [app.filament.resources.repo-resource.widgets.repos-by-team]
```

It doesn't appear to be generating a view, and I think it _should_ be. Am I missing something obvious here (I am sure I am...)?

## Details

**Command that was run:**

```
php artisan make:filament-widget ReposByTeam --resource=RepoResource
```

<img width="1210" alt="image" src="https://github.com/filamentphp/filament/assets/1449463/efb6cdd5-2abc-4140-b77e-0270d9528b6f">

**RepoResource relevant info:**

```php
    public static function getWidgets(): array
    {
        return [
            ReposByTeam::class
        ];
    }
```

**List Page relevant info:**

```php
    protected function getHeaderWidgets(): array
    {
        return [
            ReposByTeam::class
        ];
    }
```

**Comment by chaseconey** [comment]
Still seem to be experiencing this, even in the latest version (`v3.2.92`)

**Comment by chaseconey** [comment]
Finally figured it out. It was an obvious mistake, but for anyone else that finds this...make sure you clear any filament cache you may have 😬 .

```
php artisan filament:clear-cached-components
```

**Reply by wightspace** [reply]
Thank you

**Reply by YordanisPonce** [reply]
thank you!!!!
